week-1/Homework/mandatory/4-tax.js

```diff
@@ -5,7 +5,15 @@
   Sales tax is 20% of the price of the product
 */
 
+<<<<<<< Updated upstream
```

Comment:
It looks like something weird happened with git here - You probably want to make sure to edit your master branch to make sure this doesn't have these <<>>> markers in it - we can talk about it on slack if you need a hand :)